src/app/dim-ui/DestinyTooltipText.tsx

```diff
@@ -11,8 +12,9 @@ export function DestinyTooltipText({ item }: { item: DimItem }) {
   }
   return (
     <>
-      {!isDeepsightTooltip &&
```

Comment:
Yeah that's a whoops, since it didn't actually *call* the function. I had hoped our linter would catch this sort of thing.

Reply:
Yeah I ran into the same problem already once. There's an open suggestion on the TS repo https://github.com/microsoft/TypeScript/issues/45667